cmd/acr/annotate.go

```diff
@@ -288,9 +309,9 @@ func annotateUntaggedManifests(ctx context.Context,
 	// Contrary to getTagsToAnnotate, getManifests gets all the manifests at once.
 	// This was done because if there is a manifest that has no tag but is referenced by a multiarch manifest that has tags then it
 	// should not be annotated.
-	manifestsToAnnotate, err := common.GetUntaggedManifests(ctx, poolSize, acrClient, repoName, true, nil, dryRun)
+	manifestsToAnnotate, err := common.GetUntaggedManifests(ctx, poolSize, acrClient, repoName, true, nil, dryRun, includeLocked)
 	if err != nil {
-		return -1, err
+		return -1, 0, err
```

Comment:
It seems like the skipped value is always 0 for this one? Is there a reason to add this return?

Reply:
Yup, you are correct, my mistake